vignettes/trialplanning.Rmd

```diff
@@ -112,13 +111,13 @@ StudyforAvg <- getClinicalTrials(
   dropout = list(rate = 0.05, time = 12), accrual = list(param = "intensity", value = 25)
 )
 
-averageHR <- lapply(StudyforAvg, avergHR)
-MeanAverageHR <- mean(unlist(averageHR))
+averageHR <- sapply(StudyforAvg, avergHR)
+MeanAverageHR <- mean(averageHR)
 ```
 
+In our example, we get an average OS HR of `r round(MeanAverageHR, 3)`.
```

Comment:
This is the average over all trials? For illustration would it not be enough to simply simulate one trial and compute the AHR?

Reply:
that is how it was before, yes. @niessl what do you think?

Reply:
I took the average over several trials to get a more robust estimate